frontend/packages/topology/console-extensions.json

```diff
@@ -5,5 +5,145 @@
       "scope": "devconsole",
       "reducer": { "$codeRef": "reduxReducer" }
     }
+  },
+  {
+    "type": "console.topology/details/tab",
+    "properties": {
+      "id": "topology-side-bar-tab-details",
+      "label": "%topology~Details%"
+    }
+  },
+  {
+    "type": "console.topology/details/tab",
+    "properties": {
+      "id": "topology-side-bar-tab-resource",
+      "label": "%topology~Resources%"
+    }
+  },
+  {
+    "type": "console.topology/details/tab-section",
+    "properties": {
+      "id": "topology-tab-section-deployment-config",
+      "tab": "topology-side-bar-tab-details",
+      "section": { "$codeRef": "workload.getDeploymentConfigSideBarDetails" }
+    }
+  },
+  {
+    "type": "console.topology/details/tab-section",
```

Comment:
I feel these could be contributed by `@console/app` instead of topology package. We contribute actions related to these resources from console app.